### DISCUSSION TITLE
如何对正文前内容编罗马数字页码

### DISCUSSION THREAD

**Discussion by zhenglongjiepheonix** [sjtug/SJTUThesis#650] [提问] [answered]
为啥frontmatter显示不出罗马数字编号

**Comment by skyzh** [comment]
请补充一点详细信息。硕士/本科模版？具体哪个位置需要显示罗马数字？

**Reply by zhenglongjiepheonix** [reply]
就是本科模板，对正文前的内容进行编号

**Comment by skyzh** [comment]
<img width="873" alt="image" src="https://user-images.githubusercontent.com/4198311/119592775-aeb5cd80-be0b-11eb-936f-5420926f535b.png">

```
\ctexset{
    section/number = \Roman{section}
}
```

**Comment by skyzh** [comment, marked as answer]
`sjtusetup.tex` 中可以设置。

```tex
  style = {
    frontmatter-numbering = true
  },
```

**Reply by zhenglongjiepheonix** [reply]
好了，感谢

**Reply by AlexaraWu** [reply]
《撰写规范》要求「中英文摘要不需编页码」，所以本科模板默认前置内容是没有页码的，但是留了一个选项。

**Reply by skyzh** [reply]
这个提问我读了至少 5 遍，每次都有新猜想，现在才知道原来问的是罗马数字页码……
